### PR TITLE
Dance page - sponsor logo with text 

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to display amazon log with text.

### Implementation
- Imported sponsor logo view partial
- Add I18n strings
- Change font size of section title heading
- Deleted place holder for logo

### Next
- [ ] Display videos
- [ ] Update tutorial section with new background image

Future work includes: 
- [ ] i18n for hardcoded strings for each section
- [ ] creating view partials for each section
- [ ] Updating dance landing page skeleton with view partials for each section
- [ ] Making landing page responsive
- [ ] style tweaks when we see it in the context of the page 

screen shot 2018-11-06 at 3 29 48 pm

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  margin-top: 30px;
+  clear: both;
+  padding-top: 30px;
+}
+
+.sponsor-subtext-description {
+  font-size: 14px;
+  text-align: center;
+  padding-top: 20px;
+  margin-left: 5%;
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css:14]: The % here might get wonky when you check responsiveness on mobile; you can leave for now and adjust later.

**Comment by Erin007** [comment]
You have a note to switch the branch, but it's still comparing with staging?

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+%link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-sponsor-logo.css'}
+
+%div.sponsor-logo-with-text
+  %p= I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
+  %div
+    %img{src:"/images/fit-450/AFE-Dark-Orange-logo.png"}
+
+%div.sponsor-subtext-description
+  = I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_description_with_link, amazon_future_engineer_link: 'www.amazon.com' )
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: Have you tested the link?

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css (hunk 1 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css:14]: Good point!  I am curious as to why px work better when checking for responsiveness on mobile?

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css:14]: Let's say the full width is 1000px, and margin-left of 5% means you'd have 50px on the left of the element.  As your screen size shrinks, 5% of a smaller screen size is much smaller than the margin you had before.  This effect may or may not be what you want to have happen, so it's good to check various screen sizes.

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml (hunk 2 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: Link tested and updated! :-) Thanks

**Pull request merged**